docs/authentication/configuration/restrictions.mdx

```diff
@@ -1,91 +1,79 @@
 
-To enable this feature, in the Clerk Dashboard, go to **User & Authentication > [Restrictions](https://dashboard.clerk.com/last-active?path=user-authentication/restrictions)** and go to the **Allowlist** section. Turn on the **Enable allowlist** toggle.
+To enable this feature:
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com).
+2. In the navigation sidebar, select **User & Authentication > [Restrictions](https://dashboard.clerk.com/last-active?path=user-authentication/restrictions)**.
+3. In the **Allowlist** section, toggle on **Enable allowlist**.
 
-<Callout type="warning" emoji="⚠️">
+<Callout type="danger">
   Be aware that turning on the Allowlist feature without adding any identifier exceptions blocks *all* sign-ups.
 </Callout>
 
-<Images
-width={3024}
-height={1654}
-src="/docs/images/authentication/restrictions/allowlist.jpg"
-alt="The 'Allowlist' section in the 'Restrictions' tab under the 'User & Authentication' section of the Clerk Dashboard. A red arrow is pointing to the 'Enable allowlist' toggle, which is toggled on."
-/>
-
-After turning on the Allowlist feature, you can add individual email addresses and phone numbers, or allowlist entire email domains.
-
-For example, if you add "clerk.dev" as an allowed email domain, it means that anybody with a "@clerk.dev" email address can sign up for your application. Email addresses from different domains will not be able to sign up.
-
-<Callout type="warning" emoji="⚠️">
-  In the case you have enabled the Allowlist and the Blocklist and have added the same identifier in both, the Allowlist takes precedence.
-</Callout>
-
 ## Blocklist
 
 <Callout type="info">
   Blocklist is a premium feature and is not available on the Free plan. [Upgrade your plan](https://clerk.com/pricing) to enable this feature.
 </Callout>
 
-The Blocklist feature allows you to control who can get access to your application. It can restrict sign ups for a certain set of email addresses or phone numbers that you define.
+By adding specific identifiers to the blocklist, users with those identifiers will be blocked from signing up for your application. This is most commonly useful in attack prevention. For example, if you have spam accounts signing up with a specific email domain, you can block those email addresses or the entire email domain to prevent further sign-ups. For example, if you add `clerk.dev` as a blocked email domain, it means that anybody with a `@clerk.dev` email address will not be able to sign up for your application.
```

Comment:
```suggestion
By adding specific identifiers to the blocklist, users with those identifiers will be blocked from signing up for your application. This is useful for attack prevention, such as when multiple spam accounts sign up to your up. For example, if you add `clerk.dev` as a blocked email domain, it means that anybody with a `@clerk.dev` email address will not be able to sign up for your application.
```

Suggested removing the first example, as it feels redundant with the second one there and I think this still conveys your overall message